app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Let’s forget about `ConstraintLayout` for now, use `FrameLayout` + `LinearLayout` instead.